zebra-chain/Cargo.toml

```
@@ -44,11 +44,13 @@ sha2 = { version = "0.9.5", features=["compress"] }
 subtle = "2.4"
 thiserror = "1"
 x25519-dalek = { version = "1.1", features = ["serde"] }
+zcash_history = { git = "https://github.com/zcash/librustzcash.git"}
```

I think this space probably doesn't matter to the parser
```suggestion
zcash_history = { git = "https://github.com/zcash/librustzcash.git" }
```